integration-tests/src/test/java/oracle/weblogic/kubernetes/ItServerStartPolicy.java

```diff
@@ -1130,6 +1132,131 @@ public void testRestartingMSWithExplicitServerStartStateWhileVaryingReplicaCount
     logger.info("managed server " + serverName + " restarted successfully.");
   }
 
+  /**
+   * Scale the configured cluster using the sample script scaleCluster.sh script
+   * Verify that server(s) in the configured cluster are scaled up and in RUNNING state.
+   * Verify that server(s) in the dynamic cluster are not affected.
+   * Restore the env using the sample script stopServer.sh.
+   */
+  @Order(19)
+  @Test
+  @DisplayName("Scale the configured cluster with scaleCluster.sh script")
+  public void testConfigClusterScale() {
+    int newReplicaCount = 2;
+    String configServerName = "config-cluster-server" + newReplicaCount;
+    String configServerPodName = domainUid + "-" + configServerName;
+    String dynamicServerPodName = domainUid + "-managed-server" + newReplicaCount;
+
+    // use clusterStatus.sh to make sure the server-to-be-test doesn't exist
+    // String regex matches below
+    // cluster        min  max  goal  current  ready
+    // clusterName     1    5    1      1       1
+    String regex = ".*" + CLUSTER_1 + "(\\s+)1(\\s+)5(\\s+)1(\\s+)1(\\s+)1";
```

Comment:
What call returns the formatted data given above? I can see how the regex matches this, but this seems really brittle.

Reply:
I see it's from clusterStatus.sh... Okay, I guess that's what we have to test.